index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    
+    <main id="main">
+      <div class="tribute-Intro">
+        <h1 class="tribute-main-title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </div>
+      <div class="tribute-image">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="eight men standing on a field">
+        <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+      </div>
```

Comment:
I think it would be better to  use figcaption tag for  the img  tag because as said in the instructions of the project.
Kindly add id img-div to the div too.